bin/direxec

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import os
+import subprocess
+import sys
+
+mode = os.path.basename(__file__)
+
+dir_odoo = '/opt/odoo'
+dir_common = os.path.join(dir_odoo, 'common', '%s.d' % mode)
+dir_custom = os.path.join(dir_odoo, 'custom', '%s.d' % mode)
+
+files = [
+    os.path.join(dir_common, d) for d in os.listdir(dir_common)
+]
+
+try:
+    files += [
+        os.path.join(dir_custom, d) for d in os.listdir(dir_custom)
+    ]
+except OSError:
+    pass
+
+for command in sorted(files):
+    if os.access(command, os.X_OK):
+        exit_code = os.system(command)
+        if exit_code:
+            raise EnvironmentError(
```

Comment:
Ohhhhhh wait ok reading the docs of `execvp` & then your comments again - I totally get it! Thanks for the explanations